pages/linux/dconf-read.md

```diff
@@ -0,0 +1,13 @@
+# dconf read
+
+> Read key values from dconf databases.
+> See also: `dconf`.
+> More information: <https://manned.org/dconf>.
+
+- Print a specific key value:
+
+`dconf read {{/path/to/key}}`
```

Comment:
```suggestion
`dconf read {{path/to/key}}`
```

Reply:
Without leading slash I obtain: `error: dconf key must begin with a slash` for smtl like `dconf read org/mate/panel/toplevels/top/background/color`.

Reply:
Oh! Got it.